example/hsdis/hsdis.c

```diff
@@ -70,7 +70,7 @@
 #define ADDR_FORM	FCML_OM_32_BIT
 #endif
 
-char HELP[] = "Optional arguments:\n"
+__attribute__((visibility("default"))) char HELP[] = "Optional arguments:\n"
```

Comment:
This might cause issues with non-GCC, let me know if this is a problem and I’ll rework the patch.